packages/tests/server/interop/inference.test.ts

```diff
@@ -204,7 +204,7 @@ describe('infer subscription input & output', () => {
       },
     })
     .interop();
-  type TSubscriptions = typeof router['_def']['subscriptions'];
+  type TSubscriptions = (typeof router)['_def']['subscriptions'];
```

Comment:
Where are these changes coming from? Updated prettier or lint?

Reply:
Answered in the comment above :)

> Parentheses changes are coming from prettier@2.8.2 - https://github.com/prettier/prettier/blob/main/CHANGELOG.md#add-parentheses-for-tstypequery-to-improve-readability-14042-by-onishi-kohei